src/Powercord/plugins/pc-announcements/index.js

```diff
@@ -31,16 +30,21 @@ module.exports = class Announcements extends Plugin {
               getModule([ 'selectGuild' ]).selectGuild(GUILD_ID);
             });
           }
-        }
-      }, true);
+        },
+        alwaysDisplay: true,
```

Comment:
eslint won't like this `,` 👀
```suggestion
        alwaysDisplay: true
```